typescript/packages/browser-lite/package.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "@0x/mesh-browser-lite",
+    "version": "8.2.0",
+    "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
```

We should explain the difference between the two packages in the description. Something like:

```suggestion
    "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser. To use this package, you must serve your own copy of the Mesh WebAssembly bytecode.",
```